src/averageAges.js

```diff
@@ -15,11 +15,19 @@
  * @return {number}
  */
 function calculateMenAverageAge(people, century) {
-  // write code here
-  // learn how to use array methods like .filter .map .some .every .find .reduce
-  // avoid using loop and forEach
-  // replace `if ()` statement with &&, || or ?:
-  // without nesting
+  let men = people.filter(person => person.sex === 'm');
+
+  men = century ? men.filter(person => {
+    return Math.ceil(person.died / 100) === century;
+  }) : men;
```

Comment:
```suggestion
  men = century
    ? men.filter(person => Math.ceil(person.died / 100) === century)
    : men;
```
Відформатуй будь ласка красиво всюди